tensorflow/core/platform/platform.h

```diff
@@ -45,7 +45,12 @@ limitations under the License.
 
 // Since there's no macro for the Raspberry Pi, assume we're on a mobile
 // platform if we're compiling for the ARM CPU.
-#define IS_MOBILE_PLATFORM
+
+// TODO: Disable this define because it breaks (at least) in:
+//  ./tensorflow/core/platform/gif.h
+//  ./tensorflow/core/platform/jpeg.h
+//  ./tensorflow/core/platform/png.h
+// #define IS_MOBILE_PLATFORM
```

Comment:
Just FTR I did a build of libtensorflow.so with IS_MOBILE_PLATFORM (and added DTINT64 because SparseToDense depends on that) and there was nearly no change at runtime, besides smaller .so file.